repo2docker/app.py

```diff
@@ -342,6 +343,15 @@ def get_argparser(self):
             type=str,
             help=self.traits()['appendix'].help,
         )
+
```

Comment:
should we copy your comment from the PR about why we are checking explicitly for `--version` without parsing and yet add this arg here as a code comment so that future generations remember why things are this way?

Reply:
sure!